migrations/20241105164503_2.sql

```diff
@@ -0,0 +1,40 @@
+-- Force anything running this migration to use the right search path.
+set local search_path to underway;
+
+-- Remove attempt-specific columns from `task`
+alter table underway.task
+drop column if exists retry_count,
+drop column if exists initial_interval_ms,
+drop column if exists max_interval_ms,
+drop column if exists backoff_coefficient,
+drop column if exists error_message,
+drop column if exists started_at,
+drop column if exists succeeded_at,
+drop column if exists last_failed_at;
+
+alter table underway.task
+add column if not exists retry_policy jsonb not null default '{"max_attempts": 5, "initial_interval_ms": 1000, "max_interval_ms": 60000, "backoff_coefficient": 2.0}';
+
+alter table underway.task
+add column if not exists completed_at timestamp with time zone;
+
+alter table underway.task
+add column if not exists last_attempt_at timestamp with time zone;
+
+create table underway.task_attempt (
+    task_id         uuid not null,
+    task_queue_name text not null,
```

Comment:
i thought that task always run in the same queue?
so i think this is `task` related field. Am i missing something?

Reply:
You're right. The reason it's included is because `task` has a composite key and the queue name is therefore required to create the fk.

Reply:
ah, oh
i was sure `task_id` is uniq...